src/grt/include/grt/GRoute.h

```diff
@@ -116,6 +117,9 @@ struct CongestionInformation
   std::set<odb::dbNet*, cmpById> sources;
 };
 
+using CapacityReductionData
+    = std::vector<std::vector<std::pair<uint8_t, uint8_t>>>;
```

Comment:
Give a comment as its not obvious what the indices or the pair are.  Better still would be to use a struct with meaningful names rather than a pair.